cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxHelper.java

```diff
@@ -226,6 +284,29 @@ public static boolean openURL(String url) {
         }
         return ret;
     }
+    
+    public static long[] getObbAssetFileDescriptor(final String path) {
+        long[] array = new long[3];
+        if (Cocos2dxHelper.sOBBFile != null) {
+            AssetFileDescriptor descriptor = Cocos2dxHelper.sOBBFile.getAssetFileDescriptor(path);
+            if (descriptor != null) {
+                try {
+                    ParcelFileDescriptor parcel = descriptor.getParcelFileDescriptor();
+                    Method method = parcel.getClass().getMethod("getFd", new Class[] {});
+                    array[0] = (Integer)method.invoke(parcel);
```

Comment:
I got it, use reflection to avoid depending on API 12 because ParcelFileDescriptor.getFd() exists since API 12. But it is better to add some comment here.